actionmailbox/db/migrate/20180917164000_create_action_mailbox_tables.rb

```diff
@@ -10,4 +10,11 @@ def change
       t.index [ :message_id, :message_checksum ], name: "index_action_mailbox_inbound_emails_uniqueness", unique: true
     end
   end
+
+  private
+    def primary_key_type
+      config = Rails.configuration.generators
+      setting = config.options[config.orm][:primary_key_type]
+      setting || :primary_key
```

Comment:
I'd prefer to copy the exact code from ASt for this https://github.com/rails/rails/blob/main/activestorage/db/update_migrate/20191206030411_create_active_storage_variant_records.rb#L16-L19